config/auth.php

```diff
@@ -31,7 +31,7 @@
     | users are actually retrieved out of your database or other storage
     | mechanisms used by this application to persist your user's data.
     |
-    | Supported: "session", "token"
+    | Supported: "session", "token", 'passport'
```

Comment:
Should it be "passport" instead of 'passport'?

Reply:
Yeah - just a commented text, so it make no difference :p